docs/ATLAS.md

```diff
@@ -0,0 +1,23 @@
+### Setup Atlas using terraform ###
+
+Follow these steps to setup a M2 cluster in Atlas to be used by the uDeploy portal.
+
+1. Register an API key in Atlas.  `Atlas -> Project -> Access management -> API Keys`
+    
+    - Click `Manage -> Create API Key`
+    - Enter a short description for this key
```

Comment:
Is a description required? If so, not sure we need to document it here. That should be intuitive.